Finance/BatchList.ascx.cs

```diff
@@ -37,13 +45,51 @@ namespace RockWeb.Plugins.rocks_kfs.Finance
     [DisplayName( "Batch List With Account Filter" )]
     [Category( "KFS > Finance" )]
     [Description( "Lists all financial batches and provides filtering by account, campus, status, etc." )]
-    [LinkedPage( "Detail Page", order: 0 )]
-    [BooleanField( "Show Accounting Code", "Should the accounting code column be displayed.", false, "", 1 )]
-    [BooleanField( "Show Accounts Column", "Should the accounts column be displayed.", true, "", 2 )]
-    [CodeEditorField( "Summary Lava Template", "The lava template for display the content for summary", CodeEditorMode.Lava, CodeEditorTheme.Rock, order: 3, defaultValue: @"
+
+    [LinkedPage( "Detail Page",
+        Description = "",
+        Order = 0,
+        Key = AttributeKey.DetailPage )]
+
+    [BooleanField( "Show Accounting Code",
+        Description = "Should the accounting code column be displayed.",
+        DefaultBooleanValue = false,
+        Order = 1,
+        Key = AttributeKey.ShowAccountingCode )]
+
+    [BooleanField( "Show Accounts Column",
+        Description = "Should the accounts column be displayed.",
+        DefaultBooleanValue = true,
+        Order = 2,
+        Key = AttributeKey.ShowAccountsColumn )]
+
+    [BooleanField( "Show Only Active Accounts on Filter",
+        Description = "If account filter is displayed, only list active accounts",
+        DefaultBooleanValue = false,
+        Order = 3,
+        Key = AttributeKey.ActiveAccountsOnlyFilter )]
+
+    [AccountsField( "Accounts",
+        Description = "Limit the results to batches that match the selected accounts.",
```

Comment:
"Limit the results to batches with transactions that match the selected accounts."